packages/synapse-react-client/src/components/ChallengeTeamWizard/OpenMembershipInvitation.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useCallback } from 'react'
+import {
+  useAddMemberToTeam,
+  useGetAllOpenMembershipInvitations,
+  useGetCurrentUserProfile,
+  useGetTeam,
+} from '../../synapse-queries'
+import { UserBadge } from '../UserCard/UserBadge'
+import { SynapseSpinner } from '../LoadingScreen/LoadingScreen'
+import { Alert, Button, ButtonProps, Stack, Typography } from '@mui/material'
+import UserOrTeamBadge from '../UserOrTeamBadge'
+import { noop } from 'lodash-es'
+
+export type AcceptInvitationProps = {
+  teamId: string
+  onSuccess?: () => void
+}
+
+export function AcceptMembershipInvitationButton(
```

Comment:
Button to accept joining a team--since this doesn't actually need to share data with the `OpenMembershipInvitation` component, it can be rendered separately, as long as the same `teamId` prop is passed to both components. A bit more simple than the `submit/useImperativeHandle` pattern, which isn't necessary for this case.